roles/nova-common/tasks/logging.yml

```diff
@@ -7,3 +7,6 @@
       - missingok
       - rotate 7
       - compress
+      - sharedscripts
+    scripts:
+      - postrotate: "/usr/sbin/service nova-novncproxy reload"
```

Comment:
Nope.  Ursula deploy failed on this task.  Please reject the pull request and I will rework it.